backend/uclapi/workspaces/views.py

```diff
@@ -179,3 +179,65 @@ def get_survey_sensors_summary(request, *args, **kwargs):
     }, rate_limiting_data=kwargs)
 
     return response
+
+
+@api_view(["GET"])
+@uclapi_protected_endpoint(personal_data=False)
+def get_historical_time_data(request, *args, **kwargs):
+    api = OccupEyeApi()
+    survey_ids = request.GET.get("survey_ids", None)
```

Comment:
Should not this one be `request.GET["survey_ids"]` so that an `Exception` gets thrown when you do not specify it?

Reply:
The parameter is optional, and the code / docs handle this.